deployment/docker-compose/template.env

```diff
@@ -16,6 +16,7 @@ POSTGRES_DB=postgres
 USER1_USERNAME = "admin"
 USER1_PASSWORD = "fullaccess"
 USER1_API_KEY = "admin-key"
+USER1_CONTENT_QUOTA = 100
```

Comment:
Given we use this as admin, can we make this 1000?

Reply:
Can make it blank, which will be interpretted as infinite